terraform/environments/mlra/modules/ecs/main.tf

```diff
@@ -510,11 +529,11 @@ resource "aws_iam_role_policy_attachment" "ecs_task_ssm_access" {
   policy_arn = aws_iam_policy.ecs_task_execution_ssm_policy.arn
 }
 
-# Set up CloudWatch group and log stream and retain logs for 30 days
+# Set up CloudWatch group and log stream and retain logs for 1 year (365 days)
 resource "aws_cloudwatch_log_group" "cloudwatch_group" {
   #checkov:skip=CKV_AWS_158:Temporarily skip KMS encryption check while logging solution is being updated
   name              = "${var.app_name}-ecs-log-group"
-  retention_in_days = 30
+  retention_in_days = 365
```

Comment:
There is a big difference between retaining stuff for a month and a year - have we checked we want to retain for a year with a technical architect or lead?

Reply:
Fair point, have reverted this change, will add a skip for the check and we can discuss in the future as part of LASB-3390.

Reply:
FYI this is the rule/policy violation.
https://docs.prismacloud.io/en/enterprise-edition/policy-reference/aws-policies/aws-logging-policies/bc-aws-338